modules/lambdas/output.tf

```diff
@@ -26,6 +26,11 @@ output "get_current_archive_list_arn" {
   value       = aws_lambda_function.get_current_archive_list.arn
 }
 
+output "internal_reconcile_report_phantom_arn" {
+  description = "AWS ARN for the internal_reconcile_report_phantom lambda."
+  value       = aws_lambda_function.internal_reconcile_report_phantom.arn
+}
+
```

Comment:
should this be added to the modules/main.tf as well similar to https://github.com/nasa/cumulus-orca/blob/4d6cff91690984c4a8b947d89a2115d1e0009301/modules/orca/main.tf#L111 ?

Reply:
I believe that this is not necessary with our current API Gateway architecture.